backend/src/main/java/com/epam/todo/controller/TodoController.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.todo.controller;
+
+
+import com.epam.todo.model.ToDo;
+import com.epam.todo.service.TodoFacade;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Objects;
+
+@RestController
+public class TodoController {
+
+    private final TodoFacade todoFacade;
+
+    public TodoController(TodoFacade todoFacade) {
+        Objects.requireNonNull(todoFacade);
```

Comment:
With constructor injection you don't need to filter it out if it is null. Make sure about it it will be a spring managed bean.

Reply:
I agree. Previous mentor required it.
Done.